packaging/nnstreamer.spec

```diff
@@ -163,6 +163,10 @@ BuildRequires:	gst-plugins-bad-devel
 BuildRequires:	glib2-devel
 BuildRequires:	meson >= 0.50.0
 
+# tensorflow2-lite-custom requires scripts for rpm >= 4.9
+BuildRequires:  rpm >= 4.9
```

Comment:
Q: condition to check tf-lite2?
%if 0%{?tensorflow2_lite_support}
BuildRequires: rpm >= 4.9
...